.github/workflows/ci_nightly.yml

```diff
@@ -51,3 +51,19 @@ jobs:
           device: ${{ matrix.device.soc }}
           target: ${{ matrix.device.target }}
           toolchain: nightly
+
+    create-issue-on-failure:
```

Comment:
This doesn't need to be a completely new job I think.

Reply:
I tried to play with it but couldn't figure out how to do that without a separate job so I went with easier option :D

Reply:
IIRC, you could set [`continue-on-error`](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#jobsjob_idcontinue-on-error) and then transform your issue job into a steps that checks `if: failure()` something like that should do the trick I think